Automattic/wp-calypso
# Stats: fix page reloading after notice dismissing

## Proposed Changes

* Adds a state to allow the loading state to be triggered only on component mounted

## Testing Instructions

* Create a fresh JN site and connect
* Dismiss upgrade notice
* Ensure the whole page doesn't reload

## Pre-merge Checklist
